tests/e2e/hcp_cluster_test.go

```diff
@@ -260,4 +260,56 @@ var _ = Describe("HCP cluster testing",
 				textData = rosaClient.Parser.TextData.Input(out).Parse().Tip()
 				Expect(textData).To(ContainSubstring("Updated cluster '%s'", clusterID))
 			})
+
+		It("rosacli can show the details of HCP cluster well when describe - [id:54869]",
+			labels.Critical, labels.Runtime.Day2,
+			func() {
+				By("Get cluster description")
+				clusterDesc, err := clusterService.DescribeClusterAndReflect(clusterID)
+				Expect(err).To(BeNil())
+
+				By("Check values of description")
+				Expect(clusterDesc.Name).ToNot(BeEmpty())
+				Expect(clusterDesc.ID).ToNot(BeEmpty())
+				Expect(clusterDesc.ControlPlane).To(Equal("ROSA Service Hosted"))
+				Expect(clusterDesc.OpenshiftVersion).ToNot(BeEmpty())
+				Expect(clusterDesc.DNS).ToNot(BeEmpty())
+				Expect(clusterDesc.APIURL).ToNot(BeEmpty())
+				Expect(clusterDesc.Region).ToNot(BeEmpty())
+				Expect(clusterDesc.Availability[0]).To(HaveKeyWithValue("Control Plane", MatchRegexp("AZ")))
```

Comment:
Control Plane must be `MultiAZ`
DataPlane should be calculated from workerpool zones to decide if it is `SingleAZ` or `MultiAZ`

Reply:
Chnaged Control Plane value to be only `MultiAZ`. 
For data plane changed accepted values `SingleAZ` or `MultiAZ`. I am not sure how to calculate from workerpool zones. If only one workerpool is present then Data plane is SingleAZ and if multiple workerpool nodes are present then Data plane is MultipleAZ ?

Reply:
Added calculation of Data plane value from workerpool number of nodes.